### PR TITLE
Update Getting started instructions on README

### LINKED ISSUE #200: Adding usage instructions in getting started section in README.
**Description**
The getting started section in README has no instruction related to the usage of Vocabulary for local development projects.

**Problems solved**
Clear and concise instruction to allow developers to use /integrate vocabulary design system in their local development projects.

**Additional context**
This will help developers to try and use the Vocabulary design system with their projects. This will basically instruct user to use Vocabulary CDN link for usage.

**Want to take up?**

Yes

### PR DESCRIPTION
**Pull request by krysal**
**Fixes**

Fixes #200 by @techievivek 

**Description**

Add CDN links and a starter template.

**Type of PR**

This PR is a optimisation.

**Tests**
Copy the starter template in a new html document and see how it looks in browser. Try changing the link to the one with always last version.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Review by dhruvkb — CHANGES_REQUESTED** [review]
@krysal can we have the instructions in a separate file instead of making the `README.md` huge? I'd prefer to have two sections in the `INSTRUCTIONS.md` file:
```markdown
# Working on Vocabulary
...
# Working with Vocabulary
...
```

**Comment by krysal** [comment]
@dhruvkb I see your point, but also I would like to keep a short **Quickstart** section on `README.md` as is usual in library packages, to avoid users who want to try quickly having to browse files.
With this, then my plan is to leave CDN links on `README.md` and pass NPM instructions and "Starter template" to the `INSTRUCTIONS.md` file.

What do you think about this approach? Should I proceed this way?

**Comment by dhruvkb** [comment]
As for the huge HTML sample, it adds very little value beyond what the `link` tags added. The rest of that is boilerplate HTML not specific to Vocabulary.

**Comment by krysal** [comment]
The purpose of the template is to provide the minimum code for a functional page working with Vocabulary. I thought it would be a plus to add it as I noticed that it is a common practice in other CSS frameworks to give the example, as [Bootstrap](https://getbootstrap.com/docs/4.4/getting-started/introduction/#starter-template), [Bulma](https://bulma.io/documentation/overview/start/#starter-template), [Pure CSS](https://purecss.io/start/), etc, does. It could also help novice developers to start.

But if you still think there is no place for the example (neither in readme nor in instructions) I will remove it, it was not my intention to bloat the readme file, just to provide more useful information.

As for the rest, then should I move all "Getting started" section of `README.md` and pass it to the `INSTRUCTIONS.md` file?

**Comment by dhruvkb** [comment]
You can add the short instructions in the `README.md` file but move the HTML template to a CodePen or similar demo service.

File: README.md [hunk 1]
```diff
@@ -55,6 +57,21 @@ Import it
 
 `import '@creativecommons/vocabulary/css/vocabulary.css'`
 
+#### Use CDN link
```
Comment (dhruvkb) [inline README.md:60]: jsDelivr is another CDN that serves `npm` packages. Could you include these links too in the appropriate places in the doc?

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@creativecommons/vocabulary@1.0.0-beta.6/css/vocabulary.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@creativecommons/vocabulary/css/vocabulary.css">
```

File: README.md [hunk 2]
```diff
@@ -55,6 +57,21 @@ Import it
...
+```html
+<link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary@1.0.0-beta.6/css/vocabulary.css">
+```
+
+Or the link that always points to the latest version:
+
+```html
+<link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary/css/vocabulary.css">
+```
+
```
Comment (dhruvkb) [inline README.md:82]: Could you add a story in the Storybook say "Getting started" or "Installing" or "Setting up" (whatever seems most appropriate or whatever you like) with more verbose instructions? 

Just for an example, setting up SVGs to work when using the unpkg CDN is not very clear (@makkoncept would testify) and detailed instructions can only do good.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (krysal) [reply README.md:60]: Done. Hope the format is ok.

File: README.md (hunk 2 above)
Comment (krysal) [reply README.md:82]: Story added. I managed to work with the unpkg CDN and SVG files locally and upload an example to [GitHub pages](https://krysal.github.io/vocabulary-playground/index). Not sure if I am missing some instructions to explain how to use CDN links, please indicate which one, if any.

**Review by dhruvkb — APPROVED** [review]
With more descriptive content, it's good to go.

**Comment by brylie** [comment]
@all-contributors please add @krysal for docs

**Pull request merged**